Update and fix sizing for hubble data dicts

chowbao opened the pull request:
Updates and table formatting and organization changes for Hubble data dicts

chowbao commented:
* Data dicts content has been copied over
* Table css simplified and formats the table a little better

stellar-jenkins commented:
**Preview is available here:**<br>http://developers-pr1094.previews.kube001.services.stellar-ops.com

sydneynotthecity reviewed (commented):
nit: can we drop the Primary Keys from all of the Table Metadata? Most of these tables do not have primary keys, and I'd like consistency across all tables. 

Also the `history_assets` table is missing Natural Keys if you don't mind filling those in

chowbao commented:
> nit: can we drop the Primary Keys from all of the Table Metadata? Most of these tables do not have primary keys, and I'd like consistency across all tables.
> 
> Also the `history_assets` table is missing Natural Keys if you don't mind filling those in

Updated in f1268f44e77c7f8c52ace97f04d5fed909d5e0d9 and 4f192054342095a294579af88d539cf182ae4ee5

There were actually a lot of issues with the table metadata lol
* Reduced the natural keys to actual unique identifier of data by using `closed_at`
* Fixed the dbt docs link to use the public dbt docs instead of the private one
* Fixed incorrect partition fields
* Fixed cluster ordering

stellar-jenkins commented:
**Preview is available here:**<br>http://developers-pr1094.previews.kube001.services.stellar-ops.com

stellar-jenkins commented:
**Preview is available here:**<br>http://developers-pr1094.previews.kube001.services.stellar-ops.com

sydneynotthecity reviewed (approved):
Nothing blocking the merge, looks good

chowbao commented on docs/data/hubble/data-catalog/data-dictionary/history-trades.mdx:
```diff
@@ -1,34 +1,50 @@
+
+| Property | Configuration |
+| --- | --- |
+| Natural Key(s) | history_operation_id, order, ledger_closed_at |
```
Ah that's right

stellar-jenkins commented:
**Preview is available here:**<br>http://developers-pr1094.previews.kube001.services.stellar-ops.com
